lib/rules/one-dependency-per-line.js

```diff
@@ -77,16 +83,71 @@ module.exports = function (context) {
         return settings[setting] === "never";
     }
 
+    function getFileIndentationLevel() {
+        var functionLine = lineNum(args[1]);
+        var firstStatementLine = lineNum(args[1].body.body[0]);
+
+        if (functionLine !== firstStatementLine) {
+            return args[1].body.body[0].loc.start.column;
+        }
+        return 0;
+    }
+
+    function report(nodeOrToken, message) {
+        context.report({
+            node: nodeOrToken,
+            message: message,
+            fix: function (fixer) {
```

Comment:
And then instead of needing a separate `report` function, you can just pass the fix function in as needed below.